Makefile

```diff
@@ -1,14 +1,26 @@
-cs:
+phpcs:
 	php-cs-fixer fix --verbose
 
-cs_dry_run:
+eslint:
+	npm run lint -- --fix
```

Comment:
> -- --fix is correct?

Yes. `npm run lint -- --fix` tells npm to run the "lint" script from the package.json. so the resulting command is `./node_modules/.bin/eslint "--fix"`